src/lib/profiles/data-management/Current/TraitData.cpp

```diff
@@ -88,6 +88,75 @@ WEAVE_ERROR UpdateDictionaryDirtyPathCut::CutPath (PropertyPathHandle aPathhandl
     return err;
 }
 
+WEAVE_ERROR TraitSchemaEngine::ParseTagString(const char *apTagString, char **apEndptr, uint8_t& aParseRes) const
+{
+    WEAVE_ERROR err = WEAVE_NO_ERROR;
+
+    VerifyOrExit(apTagString != NULL, err = WEAVE_ERROR_INCORRECT_STATE);
+    VerifyOrExit(*apTagString == '/', err = WEAVE_ERROR_INVALID_TLV_TAG);
```

Comment:
Again, I'd go with `WEAVE_ERROR_INVALID_ARGUMENT` -- there is no invalid TLV tag involved yet.

Reply:
Done, thanks